### PR TITLE
Add id param to be hable to handle mongo's _id

### PR DESCRIPTION
**Pull request by nusson**
Please try this, I've juste add 3 lines to add an id config
so now you can change your id with "_id" to simulate a mongodb

### CODE REVIEW

**Comment by GrafGenerator** [comment]
@nusson Could you please add the test case for this?
Also, it will be good to add an example of usage, what do you think about it?

**Comment by nusson** [comment]
It's done and tested :)
forget about the other one, I messed up.

File: test/test.js [hunk 1]
```diff
@@ -79,5 +80,21 @@ describe('Server', function(){
...
+					.expect(db.mongopost[0])
+					.expect(200, null);
+			});
+		});
+		
+		it('should return 404 for mongopost if default id option' , function(done){
+			startHelper({ data: db }, null, done, function(request){
+				return request.get('/mongopost/1')
+					.expect(db.mongopost[0])
+					.expect(200, null);
```
Comment (GrafGenerator) [inline test/test.js]: I guess there should be only `.expect(404, null)`, right?:)

**Comment by GrafGenerator** [comment]
@nusson, can you please re-target pull request to develop branch? I will need to do some pre-release actions, and then merge to master. I guess you will need to close this one and open new PR with target branch 'develop', and merge develop into your branch since there were changes in develop (i changed the readme, it's looks shorter now, and you may put example of _id into the section about custom server options).

**Pull request closed without merging**